ui/admin/tests/acceptance/auth-methods/selectors.js

```diff
@@ -102,8 +102,6 @@ export const MANAGE_DROPDOWN_MAKE_PRIMARY =
 export const MANAGE_DROPDOWN_DELETE =
   '[data-test-manage-auth-method] ul li:last-child button';
 
-export const TABLE_FIRST_ROW_ACTION_DROPDOWN =
-  'tbody tr:first-child td:last-child button';
 export const TABLE_FIRST_ROW_ACTION_FIRST_ITEM =
```

Comment:
You might wonder why I didn't move `TABLE_FIRST_ROW_ACTION_FIRST_ITEM` either since it has the same name and functionality as the selector in storage-buckets?
```
export const TABLE_FIRST_ROW_ACTION_FIRST_ITEM =
  'tbody tr:first-child td:last-child ul li:first-child a';
```

This one is using a `button` while the one in storage-buckets is using an `a` tag. We may need to refactor that dropdown item but potentially not either. I decided to not include that in this PR to prevent scope creep and introduce changes outside of tests.

Reply:
Ahhh that's a great catch!! and a good solution 😉 +1 to refactor that, also totally fine with a separate PR for it. Great work 🙌